docs/data_schemas.md

```diff
@@ -203,3 +203,27 @@ message RankingExample {
 }
 
 ```
+
+## Databases
+
+Open-Assistant uses two databases, one for the backend and one for the frontend.
```

Comment:
there is actually a third one for the bot which is a small sqlite db. So backend and frontends all have their own persistent state.